plugins/content-manager/src/main/java/com/wazuh/contentmanager/ContentManagerPlugin.java

```diff
@@ -44,7 +46,8 @@
 import com.wazuh.contentmanager.utils.SnapshotManager;
 
 /** Main class of the Content Manager Plugin */
-public class ContentManagerPlugin extends Plugin implements ClusterPlugin {
+public class ContentManagerPlugin extends Plugin
+        implements ClusterPlugin, ActionPlugin, ReloadablePlugin {
```

Comment:
I could remove `ActionPlugin, ReloadablePlugin` and everything kept working.

Reply:
Fix applied 2e0e4a9cb2cc1c40d0546c20cb30e613f7ff31ef